docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -24,6 +24,8 @@ To activate your integration to OpenProject in Nextcloud, navigate to the built-
 
 Once the OpenProject integration app is downloaded and enabled, you can access it from within the _Settings_ page, via the side menu. Start by entering the _Host URL_ of your desired OpenProject instance.
 
+> **Important**: If the OpenProject host-name is `localhost`, part of a known local top-level domain (e.g. 'local', 'intranet','internal','private','corp','home','lan') or resolves to the local IP ranges `100.64.0.0/10` or `192.0.0.0/24` NextCloud will refuse to connect to it. To be able to use a "local" OpenProject server the `allow_local_remote_servers` setting needs to be set in NextCloud. For that edit the file `config/config.php` and add the parameter `'allow_local_remote_servers' => true,` to the existing `$CONFIG` array, or set the setting using the occ command: `php occ config:system:set allow_local_remote_servers --value 1`
```

Comment:
I'd rather use a lvl 4 heading, instead of a block quote.

In addition:
those top level domains are just convention. We could document nextcloud behavior here, but this can easily go out of sync, if someone changes NC behavior. We could instead write something like:

```
If your OpenPorject is hosted locally, it might be possible, that Nextcloud detects that by some defined rules. In that case, NextCloud will refuse ...
```

Reply:
> I'd rather use a lvl 4 heading, instead of a block quote.

the complete text as heading or just `Important`?

Reply:
just the important

Reply:
I've addressed your comments, please review again